### DISCUSSION TITLE
Welcome to TexTeller Discussions!

### DISCUSSION THREAD

**Discussion by OleehyO** [OleehyO/TexTeller#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YogeLiu** [comment]
Hello, this work is very greatful. I want to know if you opento the dataset? Or if can provide the detail about the dataset,for example the class(math: matrix, simple formula, complex formula, linear algebra, etc.)

**Reply by OleehyO** [reply]
Hello, the dataset will be open-sourced after the paper is published, which we expect to be after August. The data has been crawled from arXiv, so it includes a variety of formula types.

**Reply by YogeLiu** [reply]
Thanks for your reply. 
I get. Will you continue train the model in the future? I found this version model has issue for some complex formula, for example, mutli-line matrix. 
![WeChat3424003d8886545a9d86a20f71bcee0f](https://github.com/OleehyO/TexTeller/assets/56834120/3d5af4b7-9143-4e65-9e3d-4a010ccb7c16)

**Reply by OleehyO** [reply]
Yes, we will consider further increasing the amount of data and extending the context window length in the future. However, you can first try fine-tuning using images of pure formulas to see if better results can be achieved.

**Reply by YogeLiu** [reply]
ok, I see. Can you share the training detail, how long it takes and the configure of GPU?  If the paper is publish, can share the doi or url?

**Reply by OleehyO** [reply]
The link will be posted on the project homepage after the paper is published.

Regarding the training configuration, TexTeller 3.0 was trained on an A800 for 2,688 hours.
